test/rsa_sp800_56b_test.c

```diff
@@ -440,17 +440,19 @@ static int test_invalid_keypair(void)
 
 static int keygen_size[] =
 {
-    2048, 3072
+    2048, 3072, 4096
```

Comment:
Do we actually want this? It is fine on contemporary machines however on some old cpus... What does it check?

Reply:
I can take it back out, I just wanted to check the tables didnt break anything..

Reply:
Taken out,,